cmd/alert-healer/healer.go

```diff
@@ -87,19 +95,35 @@ func (b *HealerBuilder) Build() (h *Healer, err error) {
 	// Allocate the healer:
 	h = new(Healer)
 
-	// Save the references to the OpenShift client:
-	h.client = b.client
+	// Save the references to the clients:
```

Comment:
Sorry about the nitpick-
This method can be rewritten as: 
```
return &Healer{b.client, b.k8sClient, b.osClient}
```
It isn't necessary to allocate an empty object using `new(Healer)` when you have the correct fields in place.

Reply:
That is true. I would use field names as well, to make it clearer and avoid depending on the order and number of fields:

```go
return &Healer{
  k8sClient: b.k8sClient,
  osClient: b.osClient,
  informerFactory: b.informerFactory,
}
```

As you can see this method used to be much longer, and did other things that couldn't have been done directly using that style: re-factoring would have been a bit more complicated.

Note also that the allocation will happen anyhow, as the pointer escapes the function enough that the compiler will have to do the allocation anyhow.

So, I have an slight preference for the style that I used, but I can change it if you think it is important :-) .

Reply:
@jhernand  Yeah I'm just thinking how to not overload the code. Also, if the `Healer` is coupled with the two clients it's worth considering losing the setter functions - see comment bellow.